examtool/api/scramble.py

```diff
@@ -6,6 +6,7 @@ def scramble(email, exam, *, keep_data=False):
 
     def scramble_group(group, substitutions, config, depth):
         group_substitutions = select(group["substitutions"])
+        group_substitutions.update(select_no_replace(group["substitutions_match"] if "substitutions_match" in group else []))
```

Comment:
`group.get("substitutions_match", [])`, please

Reply:
Rip forgot about that.